### ISSUE TITLE
A11y: Checkout page - Headlines have incorrect semantic structure

### ISSUE THREAD

**Issue by tieupmedia** [shopware/platform#5226]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.6.7  | *Affected area / extension:* Platform(Default) 

 ----

*_Expected behaviour:_*

Headlines have a correct semantic structure over the whole page:

- Versandinformationen
  - Login
  - Gastbestellung oder Registrierung
    - Persönliche Informationen
    - Ihre Adresse (headline tag is added inside `legend`)
- Zusammenfassung
- Warenkorb

*_Actual behaviour:_*

1. The semantic structure of the headlines on the cart page isn't correct. "Zusammenfassung" and "Warenkorb" are related as childs to "Persönliche Informationen", see screenshot.
2. The login isn't accessible via semantic oriented navigation helpers (e.g. browser plugins) cause there is a (perhaps visually hidden) headline missing.
3. Be aware, that "Persönliche Informationen" and "Ihre Adresse" cannot be siblings of "Login" by their own, cause both are a inseparable unit beside the login.
4. The current structure doesn't match to the optical ![Image](https://github.com/user-attachments/assets/e5bd4b61-eab9-4524-b7fa-63a6451a2bf3)
.

*_How to reproduce:_*

1. Go to the default storefront and open the cart. Got to the checkout.
2. Inspect the source code by markup review or analyze hierarchy with a helper tool like the browser plugin "HeadingsMap".

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39193**



[created from NEXT-39193, comment 505790]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-39193, comment 525295]
